packages/fether-react/src/utils/estimateUsd.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Estimate ETHUSD
+ */
+
+export const estimateUsd = () => {
+  return fetch('https://api.etherscan.io/api?module=stats&action=ethprice')
```

Comment:
There's no decentralized way to get that, right?

Reply:
To "decentralize" it we could use several sources.

Reply:
Found the following:
- https://min-api.cryptocompare.com/pricing with 100K calls a month for free
- https://www.cryptonator.com/api (no limit ?)

Reply:
I'm only half-convinced about this.
1. It's centralized. The sole purpose of Fether is to decentralize stuff.
2. Perhaps it concerns just me. One important narrative about light clients is that we don't store any user data; our privacy policy would literally be `We don't store your data.` If we use this, then some data (IP) will be accessible by these 3rd parties.

I'm willing to sacrifice UX for these 2 points.

Reply:
I'm fine with not having this for now, and until this becomes prio, we can research better on clever ways to do this (with DEX maybe?)

Reply:
AFAICT Coinmarketcap pretty much just pulls prices from as many exchanges are registered with them, and exposes the average price across exchanges they're aware of in their API - so we could "decentralize" by doing the same. the drawback is, of course, the question of who decides which exchanges' price data get factored in and figuring out how much of the overall market volume is being taken into consideration at any given point. 

+1 on shelving this for now.